app/controllers/users_controller.rb

```diff
@@ -7,7 +7,7 @@ class UsersController < ApplicationController # rubocop:todo Style/Documentation
   authorize_resource
 
   def show
-    @printer_list = BarcodePrinter.alphabetical.where(barcode_printer_type: BarcodePrinterType96Plate.all)
+    @printer_list = BarcodePrinter.alphabetical.where(barcode_printer_type: BarcodePrinterType96Plate.all).map(&:name)
```

Comment:
Use `.pluck(:name)` This will:

- Optimise the select query to only bring back the name field
- Avoid instantiating a `BarcodePrinter` for each record, and just create strings.

Not only can this be substantially faster, but it also reduces memory footprint.